lib/widgets/matrix.dart

```diff
@@ -351,13 +359,13 @@ class MatrixState extends State<Matrix> with WidgetsBindingObserver {
           ),
         );
 
-        if (state != LoginState.loggedIn) {
-          navigatorContext?.go('/rooms');
-        }
+        navigatorContext?.go('/rooms');
```

Comment:
what if log in failed ?

Reply:
it is the case for logout with multiple clients, so if it fails it should redirect to room with another client. btw I didn't change anything on this part just removed the useless condition